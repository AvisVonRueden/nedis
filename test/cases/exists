*2
$6
EXISTS
$3
foo
--------
*3
$3
SET
$3
foo
$3
bar
--------
*2
$6
EXISTS
$3
foo
