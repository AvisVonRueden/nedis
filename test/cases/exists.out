:0
+OK
:1
:1
